wagtail_airtable/templates/wagtail_airtable/airtable_import_listing.html

```diff
@@ -1,32 +1,35 @@
 {% extends "wagtailadmin/base.html" %}
-{% load wagtailadmin_tags %}
+{% load i18n wagtailadmin_tags %}
```

Comment:
suggestion: leave i18n for a separate PR, or make use of `trans`/`blocktrans` for all the strings

Reply:
Removed i18n in https://github.com/wagtail/wagtail-airtable/pull/56/commits/5836520a366014002972ba7d5f7ba4ac8f7c5621